### PR TITLE
alumni section is ready!

### PR DESCRIPTION
**Pull request by S-BEE**
kindly review this page. 
alumniSnip
@joyeshonubi  and @Involut6

### CODE REVIEW

File: src/app/alumni/img/1.jpg
Comment (Oladunjoye) [inline src/app/alumni/img/1.jpg:1]: Can we move all the images to the public/images directory for uniformity

File: src/app/components/navigation/Navbar.js [hunk 1]
```diff
@@ -1,90 +1,75 @@
+"use client";
+import React from "react";
 import {
```
Comment (Oladunjoye) [inline src/app/components/navigation/Navbar.js:3]: I'm not sure what these changes are, but if they are independent of the alumni page, they should be in a separate PR

File: src/app/page.js [hunk 2]
```diff
@@ -5,13 +5,15 @@ import { Donation } from "./components/home/Donation";
...
 import Community from "./components/home/Community";
+import Alumni from "./alumni/page";
 
 const Home = () => (
   <Box width="full">
     <HeroSection />
     <Donation />
     <ApplyNow />
     <Ourprograms />
+    <Alumni/>
```
Comment (Oladunjoye) [inline src/app/page.js]: This might be the wrong placement. The Alumni page should be a standalone page and not a subsection of the Homepage

File: src/app/alumni/page.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            <Stack spacing={{sm:'25px', lg:'50px'}} > 
+                <VStack w={{base:'300px', lg:"500px"  }} alignItems="start"  >
+                    <Heading>Our Alumni</Heading>
+                    <Text fontSize={{base:"14px", lg:"24px"}}    >We teach based on what Allah and his prophet commanded.</Text>
+                </VStack>
+                <Stack  >
+                    <Heading>Thaanawiyyah</Heading>
+                    <Box spacing={10}>
+                        <Text fontSize={{base:"14px", lg:"24px"}} >Class of 2021</Text>
+                        <SimpleGrid spacing='20px' minChildWidth='200px' >
```
Comment (Oladunjoye) [inline src/app/alumni/page.js]: referring back to the design here, the maximum number of pictures on a row in the desktop view should be 4. Also the width/height looks like 250px at least

File: public/images/mumtiaImages/eight.jpg
Comment (Oladunjoye) [inline public/images/mumtiaImages/eight.jpg:1]: Thanks for providing these images. I'd push on the group to get the actual images we need so we don't need duplicate efforts

File: photos/photo.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
+{
```
Comment (Oladunjoye) [inline photos/photo.json]: this JSON data is duplicated in another component. let's consolidate it and export it from here. 
nit: the folder name should be utils or data

**Review by Oladunjoye — CHANGES_REQUESTED** [review]

**Comment by Oladunjoye** [comment]
General comments
1. Thanks a lot for the push on this page. WWe appreciate it and we're almost there !
2. The alumni page by default will live on its on path. SO if you navigate to /alumni, you should see it automatically.
3. Some of the data refers to subjects instead of alumni, let's remove that
4. Let's also double-check the spacing/paddings from the design between each element. The design is the source of truth
5. Can we update the proof with desktop and mobile when updated

File: photos/photo.json (hunk 4 above)
Comment (S-BEE) [reply photos/photo.json]: i removed the JSON data since its only a dummy data

File: src/app/alumni/page.js (hunk 3 above)
Comment (S-BEE) [reply src/app/alumni/page.js]: well noted. thanks for the feedback

**Review by Involut6 — COMMENTED** [review]
The image circles are not perfect circles, you may want to assign fixed width and height to them

**Review by Involut6 — APPROVED** [review]

**Review by Oladunjoye — APPROVED** [review]
I didn't see updated proof, but i trust that the page is pixel-perfect. Great job !
